x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -49,15 +50,12 @@ export const getFieldItemsData = ({
       if (categoryBrowserFields.length > 0) {
         fieldItemsAcc.push(
           ...categoryBrowserFields.map(({ name = '', ...field }) => {
-            if (!showDescriptionColumn && !!field.description) {
-              showDescriptionColumn = true;
-            }
             return {
               name,
               type: field.type,
-              description: field.description ?? '',
+              description: (EcsFlat as Record<string, EcsMetadata>)[name]?.description ?? '',
               example: field.example?.toString(),
-              category: categoryId,
+              category: name?.split('.')?.[0] ?? '(unknown)',
```

Comment:
why did we changes that?

Reply:
that's why we do not have `base` category, correct?

Reply:
Yeah so a few caveats here (cc: @yctercero ) there is no `base` category in that ecs const. Not sure if this is something we should ask ECS about but yeah no more `base` unfortunately. It makes things a little bit dirtier because some categories become `_id`, `_index` etc which is certainly not pretty.

Reply:
I think it'd be more a product question as to whether they care about this "new" form of categorization for those fields?

cc @peluja1012

Reply:
Fixed here: bd7f1214645de08ead7a9068c9af01a10324a00b